DataFormats/Scouting/interface/Run3ScoutingParticleV2.h

```diff
@@ -0,0 +1,102 @@
+#ifndef DataFormats_Run3ScoutingParticleV2_h
+#define DataFormats_Run3ScoutingParticleV2_h
+
+#include <vector>
+
+//class for holding PF candidate information, for use in data scouting
+//IMPORTANT: the content of this class should be changed only in backwards compatible ways!
+class Run3ScoutingParticleV2 {
+public:
+  //constructor with values for all data fields
+  Run3ScoutingParticleV2(float pt,
+                         float eta,
+                         float phi,
+                         float m,
+                         int pdgId,
+                         int vertex,
+                         float normchi2,
+                         float dz,
+                         float dxy,
+                         float dzsig,
+                         float dxysig,
+                         int lostInnerHits,
+                         int quality,
```

Comment:
If the maximum of lostIntterHIts and quality is below 255, you can use an `uint_8` (only 1 byte) instead of `int` (4 bytes)

Reply:
Good catch, that is the case. I will amend it.

Reply:
Fixed!